### PR TITLE
added sparkline-feature

### PR DESCRIPTION
**Pull request by denysM789**

### CODE REVIEW

File: src/components/CoinOverview/CoinOverview.js [hunk 1]
```diff
@@ -57,50 +57,50 @@ class CoinOverview extends React.Component {
               </tr>
             </StyledTHead>
             <tbody className="space-x-4">
-              {list.map((obj, index) => (
+              {list.map((coin, index) => (
                 <tr>
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinOverview/CoinOverview.js:61]: you need to add key 

https://reactjs.org/docs/lists-and-keys.html#basic-list-component

File: src/components/CoinOverview/CoinOverview.js [hunk 2]
```diff
@@ -57,50 +57,50 @@ class CoinOverview extends React.Component {
...
                   <td>{index + 1}</td>
                   <td className="flex pl-2 pr-4">
                     <img
                       className="object-scale-down h-8 w-8 pr-3 pb-2"
-                      src={obj.image}
+                      src={coin.image}
                       alt=""
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinOverview/CoinOverview.js:67]: add an alt

File: src/components/CoinOverview/CoinOverview.js [hunk 3]
```diff
@@ -118,23 +118,23 @@ class CoinOverview extends React.Component {
...
+                          }).format(coin.circulating_supply)}
                         </p>
 
                         <p>
                           $
                           {Intl.NumberFormat("en-US", {
                             notation: "compact",
                             maximumFractionDigits: 1,
-                          }).format(obj.total_supply)}
+                          }).format(coin.total_supply)}
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinOverview/CoinOverview.js:137]: this int function is being reused, create a separate formatter function

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/CoinOverview/CoinOverview.js (hunk 2 above)
Comment (denysM789) [reply src/components/CoinOverview/CoinOverview.js:67]: done

File: src/components/CoinOverview/CoinOverview.js (hunk 3 above)
Comment (denysM789) [reply src/components/CoinOverview/CoinOverview.js:137]: done with numerals.js

**Pull request merged**